kratos/utilities/entities_utilities.h

```diff
@@ -32,6 +33,179 @@ namespace Kratos
  */
 namespace EntitiesUtilities
 {
+    // Enum class representing different types of definitions
+    enum class DefinitionType {
+        Single,     // Single definition
+        Multiple    // Multiple definitions
+    };
+
+    /**
+     * @brief Template struct for entity identifier.
+     * @brief This struct is used to identify and retrieve entity types based on their names and definitions.
+     * @tparam TEntity The entity type.
+     */
+    template<class TEntity>
+    class KRATOS_API(KRATOS_CORE) EntitityIdentifier
+    {
+    public:
+        ///@name Type Definitions
+        ///@{
+
+        /// Geometry type definition
+        using GeometryType = typename TEntity::GeometryType;
+
+        /// Length of array definition
+        constexpr static std::size_t LengthArray = static_cast<int>(GeometryData::KratosGeometryType::NumberOfGeometryTypes) - 2;
```

Comment:
shouldn't this be 
static_cast<std::size_t> 
?

Reply:
Yep

Reply:
Yep